### ISSUE TITLE
Builder clears but doesn't build

### ISSUE THREAD

**Issue by KuishiKama** [BuildCraft/BuildCraft#2458]
Hi,

I downloaded BC 6.3.6 for MC 1.7.10. I used the "builder" several times successful in SMP but suddenly it stopped working (with blueprints and templates). Its still clearing the area like expected but it doesn't fill it up with the blocks anymore. A restart of the server didn't help.

It worked before. There is energy and the necessary blocks inside. I was reading several issues about the builder but they were old and closed. 

I use MC 1.7.10 on Windows 8.1 with Java 1.8.0 64bit and Forge for mods. 

Mods I use:
ArchimedesShips-1.7.1.jar
Atum-1.7.10-0.6.34.jar
buildcraft-6.3.6.jar
CodeChickenCore-1.7.10-1.0.4.29-universal.jar
Fossils_Archeology_Revival-1.7.10_Build-7.0.1.jar
MrCrayfishFurnitureModv3.4.7(1.7.10).jar
NotEnoughItems-1.7.10-1.0.3.74-universal.jar
Pam's HarvestCraft 1.7.10f.jar
roguelike-1.7.10-1.3.4.jar
witchery-1.7.10-0.21.2.jar
[1.7.10] Paleocraft Cambrian 1.5.2.jar

My latency isn't really good (~500ms) if that matters. I don't get any errors or warning about that (ingame or at the "game output" in launcher).

I have no Idea if there is more details necessary and I have to apologize for my english, its not my native language. 

Greets Kuishi

**Comment by hea3ven** [comment]
What are you trying to build? do you have all the materials required?, if posible post a screenshot of the requirements in the builder's gui.

**Comment by KuishiKama** [comment]
I tried to build a wall of iron blocks with glowstone in between. I used builders to make 3 parts of the wall without problems. Now I wanted to make the 4. and it stopped working. I'm pretty sure its not a lack of material or energy. It's all the same like before. The GUI looks like before too. It just removes the wrong blocks and stopps working without placing the new ones.

I made some tests:
**Every blueprint-builder I tried stopped working**. Even my tests with simple stone or dirt structures don't work on this SMP server anymore. Old and new blueprints don't work

**Region**
I tried different regions so its not chunk related I think (or somehow it affected a larger region). 

**Client**
Singleplayer on my client works well with the builder. Other clients on the same server (I just tried another one) have the same issues -> I think its server related.
I am making a backup of the world at the moment and will try to use the same server with new world to see if its the server config or the world. But the backup will take a long time (remote backup from germany to australia with a slow internet connection).
